src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -1,20 +1,38 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
 {
     internal static class EnvironmentConfigurationHelper
     {
+        private static readonly Dictionary<InstrumentationType, Action<TracerProviderBuilder>> AddInstrumentation = new()
+        {
+            [InstrumentationType.HttpClient] = builder => builder.AddHttpClientInstrumentation(),
+            [InstrumentationType.AspNet] = builder => builder.AddSdkAspNetInstrumentation(),
+            [InstrumentationType.SqlClient] = builder => builder.AddSqlClientInstrumentation()
+        };
+
         public static TracerProviderBuilder UseEnvironmentVariables(this TracerProviderBuilder builder, Settings settings)
         {
             var resourceBuilder = ResourceBuilder
                 .CreateDefault()
                 .AddService(settings.ServiceName ?? "UNKNOWN_SERVICE_NAME", serviceVersion: settings.ServiceVersion);
 
-            return builder
+            builder
                 .SetResourceBuilder(resourceBuilder)
                 .SetExporter(settings);
+
+            foreach (var enabledInstrumentation in settings.EnabledInstrumentations)
```

Comment:
I think blacklisting will be more appropriate for this scenario?

Reply:
Well, I was considering both, and came to a conclusion that it doesn't matter that much. Both have pros and cons, for now I think this is good enough.

Reply:
I mean you mostly would include all integrations and disable the ones that don't work for you (prob some rare case). I think that's why the original solution uses blacklisting. 
Otherwise you have to list all integrations to disable a single one.